src/firmware_new/boards/frankie_v1/ethernetif.c

```diff
@@ -440,7 +440,7 @@ static void low_level_init(struct netif *netif)
  *
  * @note Returning ERR_MEM here if a DMA queue of your MAC is full can lead to
  *       strange results. You might consider waiting for space in the DMA queue
- *       to become available since the stack doesn't retry to send a packet
+ *       to become availale since the stack doesn't retry to send a packet
```

Comment:
?

Reply:
as stupid as this sounds, this gets autogenerated, and does not get fixed by codespell. So there is nothing we can do aboutt his